.vscode/c_cpp_properties.json

```diff
@@ -1,18 +1,18 @@
 {
-    "configurations": [
-        {
-            "name": "Linux",
-            "includePath": [
-                "${workspaceFolder}/**",
-                "${workspaceFolder}/../fpp-vpp/vpp/**"
-            ],
-            "defines": [],
-            "compilerPath": "/usr/bin/clang",
-            "cStandard": "c17",
-            "cppStandard": "c++14",
-            "intelliSenseMode": "linux-clang-x64",
-            "configurationProvider": "ms-vscode.makefile-tools"
-        }
-    ],
-    "version": 4
+  // for use with remote VSCode inside Docker container
+  "configurations": [
+    {
+      "name": "Linux",
+      "includePath": [
+        "${workspaceFolder}/**"
```

Comment:
Q: I'm not an expert in VSCode configs, but shouldn't that include vpp sources?
They reside under `/vpp-src` in the build container

Reply:
You are right, the fact that includes are copied to /usr/include and hence found by the compiler misled me